cmd/swapcli/main.go

```diff
@@ -175,14 +178,11 @@ var (
 						Name:  "eth-asset",
 						Usage: "Ethereum ERC-20 token address to receive, or the zero address for regular ETH",
 					},
-					&cli.StringFlag{
-						Name:  flagRelayerEndpoint,
-						Usage: "HTTP RPC endpoint of relayer to use for claiming funds. No relayer is used if this is not set",
-					},
 					&cli.StringFlag{
 						Name: flagRelayerFee,
-						Usage: "Fee to pay the relayer in ETH:" +
+						Usage: "Fee to pay the relayer in ETH if you have insufficient funds to claim:" +
```

Comment:
When I wrote that phrase, I thought were not relaying if Bob already has enough ether in the account being claimed into. I need to look into it more. If we do it that way, which I don't think we are, it will make our integration testing hard, as you'll need a swapd instance with no ETH.

Reply:
yeah maybe revert it back for now? what's the current logic for relayer/vs no relayer?

Reply:
is it basically if the relayerFee is set, or if balance is zero?

Reply:
Yes, the relayer is used if this flag is explicitly set or if the maker has no eth.  See 
https://github.com/AthanorLabs/atomic-swap/pull/342#discussion_r1133194128 for more details.